activesupport/lib/active_support/inflector/inflections.rb

```diff
@@ -49,7 +49,7 @@ def add(words)
         end
 
         def uncountable?(str)
-          @regex_array.detect {|regex| regex.match(str) }
+          @regex_array.any? { |regex| str =~ regex }
```

Comment:
It's debatable which is better, here's another thread on the topic: https://github.com/JuanitoFatas/fast-ruby/pull/62 I think either is fine. Probably didn't need the patch I sent in.